test/Libraries/AnalysisTests/AnalysisTests.csproj

```diff
@@ -14,14 +14,23 @@
     <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="2.18.0.1400"/>
     <PackageReference Include="NUnitTestAdapter" Version="2.3.0" ExcludeAssets="all" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Xml" />
   </ItemGroup>
+    <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
+        <PackageReference Include="NUnit" Version="2.6.3" />
+        <PackageReference Include="NUnitTestAdapter" Version="2.3.0" ExcludeAssets="all" />
+        <Reference Include="System" />
```

Comment:
@mjkkirschner Does it mean that System libraries are just called when the Target Framework is .NET 4.8? So, I can assume then that if the Target Framework is .NET 6 doesn't work or am I wrong?

Reply:
I think these references are no longer required when targeting .net6 - I believe some of them have been moved to nuget packages - not `system` of course but some of the others.